packages/grafana-prometheus/src/language_provider.ts

```diff
@@ -199,8 +199,8 @@ export default class PromQlLanguageProvider extends LanguageProvider {
     }
   }
 
-  fetchLabelValues = async (range: TimeRange, key: string): Promise<string[]> => {
-    const params = this.datasource.getAdjustedInterval(range);
+  fetchLabelValues = async (range: TimeRange, key: string, limit?: string): Promise<string[]> => {
```

Comment:
When we were checking for the use of range from the old breaking compatibility, we were comparing the number of parameters. Now with the new parameter of `limit` we need to update our check.

We need to do a check for something other than number of parameters 🤔